Packs/SailPointIdentityNow/Integrations/SailPointIdentityNow/SailPointIdentityNow.yml

```diff
@@ -0,0 +1,707 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: SailPointIdentityNow
+  version: -1
+configuration:
+- additionalinfo: The FQDN/IP of the IdentityNow server to connect
+  display: IdentityNow Server URL (e.g. https://org.api.identitynow.com)
+  name: identitynow_url
+  required: true
+  type: 12
+- additionalinfo: Client Id for OAuth 2.0
+  display: Client Id (for OAuth 2.0)
+  hidden: false
+  name: client_id
+  required: true
+  type: 4
+- additionalinfo: Client Secret for OAuth 2.0
+  display: Client Secret (for OAuth 2.0)
+  hidden: false
+  name: client_secret
+  required: true
+  type: 4
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue: '50'
+  display: Maximum number of incidents per fetch
+  name: max_fetch
+  required: false
+  type: 0
+- defaultvalue: 3 days
+  display: First fetch time
```

Comment:
Please specify the format, you can use this if relevant: 
 First fetch timestamp (<number> <time unit>, e.g. 12 hours, 7 days)

Reply:
Updated.